python/cudf/cudf/core/column/lists.py

```diff
@@ -228,3 +230,53 @@ def len(self):
         dtype: int32
         """
         return self._return_or_inplace(count_elements(self._column))
+
+    def take(self, lists_indices):
+        """
+        Collect list elements based on given indices.
+
+        Parameters
+        ----------
+        lists_indices: List type arrays
+        Specifies what to collect from each row
```

Comment:
I think this needs to be indented a bit now?

Reply:
```suggestion
            Specifies what to collect from each row
```

Reply:
Sorry. Should have known better than to suggest this using the UI